alchemy_news/README.md

```diff
@@ -0,0 +1,152 @@
+# Integrating the Watson Developer Cloud AlchemyAPI News Service with Watson Explorer
```

Comment:
cleaned up a bit.  I included a few references to Bluemix since you can access Alchemy News through either Bluemix or the AlchemyAPI website.  The tutorials should be neutral now insofar as which approach is used.